app/config/FrontendAppConfig.scala

```diff
@@ -80,4 +80,6 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
   lazy val languageTranslationEnabled: Boolean =
     configuration.get[Boolean]("microservice.services.features.welsh-translation")
 
+  private val defaultMaxSearchResults = 100
+  lazy val maxSearchResults: Int      = configuration.getOptional[Int]("search.maxSearchResults").getOrElse(defaultMaxSearchResults)
```

Comment:
We might want to avoid putting too much in FrontendAppConfig, it may be worth splitting this out into a new config file (see PaginationAppConfig). This will help reduce the noise in this file

Reply:
Seems a little overkill, but sure. I'll change that